packages/mobile/src/components/payment-method/PaymentMethod.tsx

```diff
@@ -19,7 +19,7 @@ import { useColor } from 'app/utils/theme'
 import { SummaryTable } from '../summary-table'
 import type { SummaryTableItem } from '../summary-table/SummaryTable'
 
-import { CardSelectionButton } from './CardSelectionButton'
+// import { CardSelectionButton } from './CardSelectionButton'
```

Comment:
Let's just remove the code and revert when re-enabling?

Reply:
oki